### PR TITLE
New: Add placeholder text for smart input field

### PR DESCRIPTION
**Pull request by jmewes**
EPNG-3874

### CODE REVIEW

**Comment by jmewes** [comment]
We don't want to show the full list of Google categories. Only after he has typed in three characters we want to open the select menu.

So we need this placeholder tips to give the merchant some hint what he has to do next after the category selection dialogue has opened.

**Review by depoulo — COMMENTED** [review]
LGTM. But could you [add your attribute to the undocumented test page running on localhost:3000](https://github.com/jmewes/react-components/edit/new/placeholder-text/public/index.js#L79)?

**Comment by jmewes** [comment]
@depoulo , please have a look again.

File: README.md [hunk 1]
```diff
@@ -19,3 +19,6 @@ npm run build
 
 [react]: https://facebook.github.io/react/
 [VectorOpenStock]: https://www.vectoropenstock.com/vectors/preview/71474/cartoon-domestic-and-wild-animals-pack
+
+## Test page
+In order to try out the components you can run `npm start` and then open http://localhost:3000 in your browser.
```
Comment (depoulo) [inline README.md:24]: :bee:

**Review by depoulo — APPROVED** [review]

**Comment by depoulo** [comment]
@jmewes I just published v0.4.0

**Comment by jmewes** [comment]
Okay, thanks for the notice.

**Pull request merged**